README.md

```diff
@@ -12,100 +12,81 @@
 
-Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started.
+The goal of DLT is to make it easier and faster to build low-maintenance data pipelines.
 
-## How does it work?
+It's designed to run anywhere (Python 3.8+), so it can fit into your existing workflows or be scheduled independently (e.g. using GitHub Actions)
 
-DLT aims to simplify data loading for everyone.
+Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started.
 
+## Who is DLT for?
 
-To achieve this, we take into account the progressive steps of data pipelining:
+DLT is for data professionals who use Python to build pipelines.
+- DLT enables anyone who knows a bit of Python to build commercial-grade pipelines
+- DLT minimises maintenance requirements by design
+- The DLT community shares pipelines with each other, so frequently you'll be able to just use a pipeline created by someone else
 
-![](docs/DLT_Diagram_1.jpg)
-### 1. Data discovery, typing, schema, metadata
+## Why use DLT?
 
-When we create a pipeline, we start by grabbing data from the source.
+DLT takes the Python scripts you already write to the next level with a library you install using pip in order to create highly scalable, easily maintainable pipelines that are straightforward to deploy.
 
-Usually, the source metadata is lacking, so we need to look at the actual data to understand what it is and how to ingest it.
+If you end up creating your own pipeline in python, then create it with DLT and your loading scripts will benefit from
+- Automatic schema maintenance (i.e. schema inference, deployment, evolution, data contracts, etc.)
+- Configurable loading (e.g. append, replace, merge, etc.)
+- Configurable normalisation (i.e. decide how to unpack nested documents, specify date formats, etc.)
+- High performance loading engine (i.e. data engineering best practices like idempotent, atomic, scalable loading)
 
-In order to facilitate this, DLT includes several features
-* Auto-unpack nested json if desired
-* generate an inferred schema with data types and load data as-is for inspection in your warehouse.
-* Use an adjusted schema for follow up loads, to better type and filter your data after visual inspection (this also solves dynamic typing of Pandas dfs)
+## How does it work?
 
-### 2. Safe, scalable loading
+DLT aims to simplify data loading for the millions of people who have taken at least one Python course
 
-When we load data, many things can intrrerupt the process, so we want to make sure we can safely retry without generating artefacts in the data.
+To achieve this, we take into account the progressive steps of data pipelining:
 
-Additionally, it's not uncommon to not know the data size in advance, making it a challenge to match data size to loading infrastructure.
+![](docs/DLT-Diagram_2.png)
```

Comment:
@adrianbr can we use a better image here? I can update the diagram Mike previously created, if that is the issue

Reply:
+10. could you do that @TyDunn this DLT-Diagram_2.png makes me ask more questions/confuses me (why mention singer tabs? what is a lakehouse?